.github/workflows/pull_request.yml

```diff
@@ -89,3 +89,9 @@ jobs:
         env:
           # USER_TOKEN: "${{ secrets.USER_TOKEN }}"
           GH_TOKEN: "${{ github.token }}"
+
+  typos:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: crate-ci/typos@master
```

Comment:
@roi-amaler-starkware Can you verify that this package is OK to use?

Reply:
@stoobie Thank you for your review. You can find something in [here](https://github.com/crate-ci/typos)

Reply:
Yeah. Glad to see that.